src/app/components/tile/index.html

```diff
@@ -57,6 +57,12 @@
     >
     Populates the tile dashboard based on the <stache-code>SkyTileDashboardConfig</stache-code> object.
     </sky-demo-page-property>
+    <sky-demo-page-property
+      propertyName="settingsKey"
+      isOptional="true"
+    >
+    Specifies a unique key for the UI Config Service to retrieve stored settings from a database. You must you prefix the key with the name of your SPA. The UI Config Service applies sticky settings to tile dashboards and saves configuration settings for users. This property accepts <stache-code>string</stache-code> values. This property is for internal Blackbaud use only. For Blackbaud developers, the UI Config Service website describes how to <a href="configuration">apply sticky settings to tile dashboards</a>.
```

Comment:
"You must you..."

Reply:
Thanks for catching that. I removed the stray "you."